packages/strapi-plugin-documentation/services/Documentation.js

```diff
@@ -319,6 +319,16 @@ module.exports = {
     return `${formattedPluginName} - ${formattedName}`;
   },
 
+  // Get property : type and format
+  getProperty: function(type) {
+    const validTypes = ['string', 'integer', 'number', 'boolean'];
+    if (validTypes.includes(type)) {
+      return { type: this.getType(type) };
+    } else {
+      return { type: 'string', format: this.getType(type) };
```

Comment:
Depending on the type it may not be a `string` but an `object`.
For instance the type `dynamiczone` is an object, the type `component` can be an `object` or an `array` (depending on if it is repeatable)